app/actions/createJobAction.ts

```diff
@@ -168,7 +168,7 @@ export async function createJobAction(data: CreateJobData): Promise<CreateJobAct
       );
       notificationSent = true;
     } catch (notificationError) {
-      logger.error({ error: notificationError }, "Failed to send Podium notification");
+      logger.error({ err: notificationError }, "Failed to send Podium notification");
```

Comment:
`err` will serialize and print errors properly.

Reply:
Also, I pretty print with commands like: `npm run job-notifications:test | npx pino-pretty`

And the formatting is correct when the message is the second argument.